composer.json

```diff
@@ -17,7 +17,7 @@
         }
     ],
     "require": {
-        "php": "^7.0",
+        "php": "^7.1",
         "friendsofsymfony/user-bundle": "^2.0",
         "sonata-project/admin-bundle": "^3.1",
         "sonata-project/core-bundle": "^3.2",
```

Comment:
What about dropping support for symfony 3.1, as it is unsupported by symfony?

Reply:
Changed to SF ^3.1 (it was 3.0 before)

Reply:
I think I have to keep to 3.0 to be consistent with other bundles where that change is already applied. Not sure what to do, but seems weird to me to have some bundles on 3.0 others on 3.1 and others on 3.2. WDYT @core23 @greg0ire ?

Reply:
We should support every actively maintained version since the last LTS

Reply:
So, please change this to `3.2^`, 'cause 3.1 is unsupported https://symfony.com/roadmap?version=3.1#checker

Reply:
done @greg0ire @core23